File: summer2021/codewars/d.cpp
// stop spinning my words
#include <iostream>
using namespace std;

string Reverse(string s){
    string t="";
    for(int i = s.size()-1; i >=0; i--){
        t += s[i];
    }
    return t;
}

int main(){
    string s;
    getline(cin, s);
    if(s.find(' ')==string::npos){
        if(s.size() >= 5){
            cout << Reverse(s);
        }
    }else{
        int pos = 0;
        for(size_t i=0; i<s.size(); i++){
            pos = s.find(' ', pos);
        }
    }
       
}

File: summer2021/codewars/d.py
# spinning words
def spin_words(sentence):
    if sentence.find(' ') == -1:
        return sentence if len(sentence) < 5 else sentence[::-1]
    else:
        arr = list(sentence.split())
        for i in range(len(arr)):
            if len(arr[i]) >= 5:
                arr[i] = arr[i][::-1]
        return " ".join(arr)
s = input()

print(spin_words(s))